.github/workflows/release.yml

```diff
@@ -97,7 +95,7 @@ jobs:
 
   build-and-test:
     name: "Build and Test"
-    needs: [log-outputs-bump-version-generate-changelog]
+    needs: [log-outputs-bump-version-generate-changelog, bump-version-generate-changelog]
```

Comment:
I thought we talked about this being separate from the release. Did that change?